src/features/in-app-messenger/contextual/templates/__tests__/PreviewTitleBodyTwoButtonsModalTemplate.test.js

```diff
@@ -0,0 +1,84 @@
+import { shallow } from 'enzyme';
+import React from 'react';
+
+import PreviewTitleBodyTwoButtonsModalTemplate from '../PreviewTitleBodyTwoButtonsModalTemplate';
+
+describe('features/in-app-messenger/contextual/templates/PreviewTitleBodyTwoButtonsModalTemplate', () => {
+    const onAction = jest.fn();
+
+    const paramsConfigs = {
+        all: {
+            params: {
+                body: 'body',
+                button1: {
+                    label: 'button1',
+                    actions: 'actions1',
```

Comment:
should actions be an array?

Reply:
Yes, for testing purposes I used string to only check if correct action is being used, but I can also put the string to an array to be more precise